tools/local-presubmit

```diff
@@ -18,7 +18,7 @@ status "Testing all of the things."
 set -x
 
 ./tools/sigh webpack
-./tools/bazelisk ${BAZELRC_OPTS} test //java/... //javatests/... //src/... //particles/...
+./tools/bazelisk ${BAZELRC_OPTS} test --test_output=errors //java/... //javatests/... //src/... //particles/...
```

Comment:
This is already present in `gcb.bazelrc`. Are you adding it here for people who run this script locally?

Reply:
Good call. I remembered seeing it somewhere, but forgot where it was. Will abandon this PR.